apps/rediffusa/apps/server/scripts/mock-db.ts

```diff
@@ -0,0 +1,108 @@
+import neo4j from "neo4j-driver";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const driver = neo4j.driver(
+  process.env.DB_URI || "",
+  neo4j.auth.basic(process.env.DB_USER || "", process.env.DB_PASSWORD || "")
+);
+
+async function run() {
+  const session = driver.session();
+
+  try {
+    const result = await session.run(
+      `
+        CREATE(:Gender {name: 'MALE'}) ;
```

Comment:
Could use some expansion. Why not make it a free-form input and put some easy prefills in the UI layer for normies?

Reply:
good idea :)